source/admin_manual/mailbox_formats/mbox.rst

```diff
@@ -177,7 +179,7 @@ writing the mail to the mbox.**
 
 Only the first message contains the X-IMAP or X-IMAPbase header. The
 difference is that when all the messages are deleted from mbox file, a pseudo
-message is written to the mbox which contains X-IMAP header.
+message is written to the mbox which contains X-IMAP header. 
```

Comment:
Your first commit removes this, and second adds it back :)

Reply:
stray whitespaces .... meh ;-)